lib/active_merchant/billing/gateway.rb

```diff
@@ -200,6 +200,16 @@ def supports_network_tokenization?
         false
       end
 
+      def add_present_fields(to, from, fields)
```

Comment:
I think the `field` and `fields` methods should have a more similar name; I suggest changing this one to `add_fields_if_present`. I think that formulation is more explicit.

Reply:
I'm not sure about where these fields are located. Should the singular method also be public? It would be a little weird for an adapter to use a plural method for a single field, but maybe it's NBD.

Reply:
Sure - I like the idea of having singular/plural public methods, especially since both versions already exist.